manager-bundle/skeleton/config/config.yaml

```diff
@@ -25,6 +25,15 @@ framework:
         cookie_secure: auto
         cookie_samesite: lax
     fragments: { path: /_fragment }
+    php_errors:
+        log: true
+    uid:
+        default_uuid_version: 7
+        time_based_uuid_version: 7
+    annotations:
+        enabled: false
+    handle_all_throwables: true
+    http_method_override: false
```

Comment:
This setting has 0 influence in our ME because our `ContaoKernel` does not respect this (and also should not). But we might remove `Request::enableHttpMethodParameterOverride();` in `ContaoKernel` for 5.3 (which could be considered a BC break, not sure if we should do that but the Core itself does not rely on that feature).

Reply:
Same here: `Since symfony/framework-bundle 6.1: Not setting the "framework.http_method_override" config option is deprecated. It will default to "false" in 7.0.`

Reply:
Reverted in b2fef1c8dfe4e3fd2bf56308f136e15a93591e21.